electron/app/components/Sample.tsx

```diff
@@ -73,7 +73,7 @@ const Sample = ({
       />
     );
   };
-  const tooltip = `Path: ${sample.filepath}\nDouble-click for details`;
+  const tooltip = `Double-click for details`;
```

Comment:
I think this deserves more discussion. In particular, @tylerganter mentioned that details when hovering over images were nice to have - is this what you were referring to?

Reply:
I was just about to comment on this to justify. My reasoning is to keep it as simple as possible- users should learn that they can double-click to reveal a whole new world of experiences. From which they can see their filepaths, among other things.

My concern with the current setup is that the filepaths are very long and "double-click to learn more" doesn't catch my eye